FLAnimatedImage/include/FLAnimatedImageFrameCache.h

```diff
@@ -14,6 +14,11 @@
 @protocol FLAnimatedImageDebugDelegate;
 #endif
 
+typedef NS_ENUM(NSUInteger, FLAnimatedImagePreferFrameCacheStrategy) {
+    FLAnimatedImagePreferFrameCacheStrategyBest,
```

Comment:
Please provide comments to explain the differences between the 'best' and 'default'. or I think we should always use `best`.